requirements.txt

```diff
@@ -1,8 +1,4 @@
-# Replace the following with something like dwave-ocean-sdk>=6.4.2
-dimod==0.12.12
-dwave-networkx==0.8.14
-dwave-samplers==1.0.0
-dwave-system==1.19.0
+dwave-ocean-sdk>=6.6.0
 
 notebook~=7.0.0   
```

Comment:
Might as well lower-bound this.

Reply:
How about I change to `notebook~=7.0` consistent with [other ones](https://github.com/dwave-examples/reverse-annealing-notebook/blob/master/requirements.txt)?

Reply:
`~=7.0` makes more sense, yes.